src/css/templates/events.scss

```diff
@@ -1,6 +1,24 @@
 body.events {
-  div.events-extra-content {
-    padding-top: 0px;
-    margin-top: -24px;
+  div.header-strip {
+    margin-bottom: 0px; // to allow for the four-card-row div of featured items
```

Comment:
No units with 0 values

Reply:
Ah this is already handled in #1605, on which this PR is built. I should have targetted that as the destination branch to make the diff smaller. Amending the target now. Cheers!